lib/game.rb

```diff
@@ -0,0 +1,60 @@
+class Game
+    attr_accessor :score, :frames, :turn
+
+    def initialize
+      @score = 0
+      @turn = 0
+      @frames = []
+    end
+
+    def tenth_frame?
+        turn === 9
+    end
```

Comment:
Love all the extractions of the if else statements! Really makes this class and methods in it more re-usable, and better testable